receiver/hostmetricsreceiver/README.md

```diff
@@ -141,6 +142,46 @@ service:
       receivers: [hostmetrics, hostmetrics/disk]
 ```
 
+### Collecting host metrics from inside a container (Linux only)
+
+On Linux, the host metrics are collected from the `/proc` filesystem. 
+You likely want to collect metrics about the host system and not the container.
+This is achievable by following these steps: 
+
+#### 1. Bind mount the host filesystem
+
+The simplest configuration is to mount the entire host filesystem when running 
+the container. e.g. `docker run -v /:/hostfs ...`.
+
+You can also choose which parts of the host filesystem to mount, if you know 
+exactly what you'll need. e.g. `docker run -v /proc:/hostfs/proc`.
```

Comment:
Yes, I like that idea. Are you aware of which filesystems are used by which scrapers? From my quick look, I only saw calls to `/proc` + the filesystems that the user specifies in the filesystem scraper. Are there other ones?

Reply:
> Could we give more prescriptive advice around which host filesystems are used by which scrapers? I think the default recommendation should be to mount only the set of filesystems that gopsutil uses, and to optionally further restrict the mountpoints depending on the scrapers that are used. Mounting the entire root directory seems like it should be avoided in most cases.

I think this should be its own issue and marked for `good-first-issue` since this could be a lot of back and forth on what could be recommended.

Reply:
@evan-bradley I've created the request here https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/16093 to make sure this isn't lost.